cocos/2d/CCProgressTimer.cpp

```diff
@@ -78,7 +78,7 @@ bool ProgressTimer::initWithSprite(Sprite* sp)
     setSprite(sp);
 
     // shader state
-    setGLProgramState(GLProgramState::getOrCreateWithGLProgramName(GLProgram::SHADER_NAME_POSITION_TEXTURE_COLOR));
+    setGLProgramState(sp->getGLProgramState()/*GLProgramState::getOrCreateWithGLProgramName(GLProgram::SHADER_NAME_POSITION_TEXTURE_COLOR)*/);
```

Comment:
Any idea? I think we should not change it like this, ProgressTimer needs to use its own shader. I will send a PR to revert it.